api/api/controllers/kvdb_controller.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import logging
+from connexion import request
+from connexion.lifecycle import ConnexionResponse
+
+from api.controllers.util import json_response
+from api.util import remove_nones_to_dict, parse_api_param, raise_if_exc
+from api.models.kvdb_model import KVDBCreateModel, KVDBUpdateModel
+from wazuh import kvdb
+from wazuh.core.cluster.dapi.dapi import DistributedAPI
+
+logger = logging.getLogger('wazuh-api')
+
+
+async def get_kvdbs(pretty: bool = False, wait_for_complete: bool = False,
+                    offset: int = 0, limit: int = None, select: list = None,
+                    sort: str = None, search: str = None, q: str = None,
+                    distinct: bool = False, type_: str = None, kvdb_id: list = None) -> ConnexionResponse:
+    """List KVDB resources.
+
+    Parameters
+    ----------
+    pretty : bool
+        Show results in human-readable format.
+    wait_for_complete : bool
+        Disable timeout response.
+    offset : int
+        First element to return in the collection.
+    limit : int
+        Maximum number of elements to return.
+    select : list
+        Select which fields to return (separated by comma).
+    sort : str
+        Sort the collection by a field or fields (separated by comma). Use +/- at the beginning
+        to list in ascending or descending order.
+    search : str
+        Look for elements with the specified string.
+    q : str
+        Query to filter results by.
+    distinct : bool
+        Look for distinct values.
+    type_ : str
+        Policy type. Allowed values: 'testing' | 'production'.
+    kvdb_id : list[str]
+        Filter by KVDB IDs (e.g. kvdb_id=foo&kvdb_id=bar).
+
+    Returns
+    -------
+    ConnexionResponse
+        API response.
+    """
+    f_kwargs = {
+        'policy_type': type_,
```

Comment:
Same for the rest of the endpoints

```suggestion
        'policy_type': policy_type,
```

Reply:
The integration test environment is not working properly; yesterday it was mentioned that its validation will be postponed for later.